pkgs/games/heroic/fhsenv.nix

```diff
@@ -81,22 +82,31 @@ buildFHSUserEnv {
     libselinux
     libsndfile
     libsndfile
+    libsoup
     libtheora
     libtiff
     libusb1
     libv4l
     libva
+    libvdpau
     libvorbis
+    libvpx
+    libwebp
     libxkbcommon
     libxml2
     mpg123
     ncurses
     ocl-icd
+    openal
     openldap
+    openssl
+    openssl_1_1
```

Comment:
That looks like it could cause conflicts

Reply:
Surprisingly, Wine-GE was trying to load both versions of `libssl`.